lib/crawly/engine.ex

```diff
@@ -242,12 +242,17 @@ defmodule Crawly.Engine do
 
   defp set_spider_log(spider_name, crawl_id) do
     log_dir = Crawly.Utils.get_settings(:log_dir, spider_name, "/tmp")
```

Comment:
could we switch the hardcoded `/tmp` to a generated path, so that it will be windows friendly 

https://til.hashrocket.com/posts/8rkl48vqmc-get-a-writable-temporary-directory-in-elixir

Reply:
Done